documentation/developer-guide/modules/tooling-guide/pages/maven-plugin.adoc

```diff
@@ -523,6 +523,52 @@ Configuration Properties:
 | `outputDirectory` | The path to the directory where the generated JSON payload will be written to. | `String` | none | {ok}
 |===
 
+=== Generating Sample JSON-LD
```

Comment:
This does not create sample data for an Aspect Model, but a JSON-LD representation of the model itself.
```suggestion
=== Generating JSON-LD representation of an Aspect Model
```